### PR TITLE
Re view model

### PR DESCRIPTION
**Pull request by jrchudy**
This PR addresses issue #1096. The view model is no longer changed while transforming the view values for submission.

Also fixes a change to the makefile that I accidentally pushed to master.

### CODE REVIEW

File: recordedit/form.controller.js [hunk 1]
```diff
@@ -114,12 +115,6 @@
...
                 var col = $rootScope.reference.columns[i];
 
-                // If this column is disabled, its value is already in the format
-                // needed for submission.
-                if (col.getInputDisabled(context.appContext)) {
-                    continue;
-                }
-
                 var rowVal = row[col.name];
                 if (rowVal) {
```
Comment (RFSH) [inline recordedit/form.controller.js]: This is not related to what you changed in this PR. But the [if statement in `default`](https://github.com/informatics-isi-edu/chaise/blob/261a5780b623466ac1163264d4c28271e0b46018/recordedit/form.controller.js#L136) will never be true, since we have this if statement here.

Comment (jrchudy) [inline recordedit/form.controller.js]: That is correct. I should probably remove that line.
